supply/lib/supply/client.rb

```diff
@@ -340,6 +353,19 @@ def upload_bundle(path_to_aab)
       return result_upload.version_code
     end
 
+    def upload_bundle_to_internal_app_sharing(package_name, path_to_aab)
+      # NOTE: This Google API is a little different. It doesn't require an active edit.
+      result_upload = call_google_api do
+        client.uploadbundle_internalappsharingartifact(
+          package_name,
+          upload_source: path_to_apk,
```

Comment:
should this be `path_to_aab`?

Reply:
Good catch. Sorry about that. Just pushed a fix for that.